alot/commands/__init__.py

```diff
@@ -22,6 +22,20 @@ def __init__(self):
         self.undoable = False
         self.help = self.__doc__
 
+        mode, cmdname = reverse_lookup_command(self.__class__)
+        logging.debug("fetching commands for: %s, %s, %s",
+                self.__class__.__name__, mode, cmdname)
+
+        # fetch and set pre and post command hooks
+        get_hook = settings.get_hook
+        self.prehook = get_hook('pre_%s_%s' % (mode, cmdname)) or \
```

Comment:
Instead of using `\\` to escape the newline, wrap the right side in parens so it can be lined up:
```python
self.prehook = (get_hook('pre_%s_%s' % (mode, cmdname)) or
                get_hook('pre_global_%s' % cmdname))
```
This applies to posthook too.